src/StatusQ/Components/StatusChatToolBar.qml

```diff
@@ -88,10 +88,17 @@ Rectangle {
             type: StatusFlatRoundButton.Type.Secondary
             visible: !!statusChatToolBar.popupMenu
 
+            property bool showMoreMenu: false
             onClicked: {
-                statusChatToolBar.menuButtonClicked()
+                if (showMoreMenu) {
+                    popupMenuSlot.item.popup(-popupMenuSlot.item.width + menuButton.width, menuButton.height + 4)
+                }
                 highlighted = true
-                popupMenuSlot.item.popup(-popupMenuSlot.item.width + menuButton.width, menuButton.height + 4)
+                statusChatToolBar.menuButtonClicked()
+            }
+
+            onPressed: {
+                showMoreMenu = !showMoreMenu;
```

Comment:
Can't we check for `popupMenuSlot.item.opened` instead of introducing a new `showMoreMenu` property?

Reply:
It's explained in the commit message, the problem is that popup has a CloseOnReleaseOutside policy so it will anyway close if the user clicks in the 'more' button but since this button calls the popup function again it will re-open, hence something like this
```
                if (!popupMenuSlot.item.opened) {
                    popupMenuSlot.item.popup(-popupMenuSlot.item.width + menuButton.width, menuButton.height + 4)
                }
```
it will always open the popup.